jax/numpy/lax_numpy.py

```diff
@@ -2625,6 +2625,30 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
   return true_divide(dot(X, X_T.conj()), f).squeeze()
 
 
+@_wraps(onp.corrcoef)
+def corrcoef(x, y=None, rowvar=True, bias=None, ddof=None):
+  msg = ("jax.numpy.cov not implemented for nontrivial {}. "
+         "Open a feature request at https://github.com/google/jax/issues !")
+  if y is not None: raise NotImplementedError(msg.format('y'))
+
+  c = cov(x, y, rowvar)
+  try:
+      d = diag(c)
+  except ValueError:
+      # scalar
+      return divide(c, c)
```

Comment:
This seems strange. Isn't this always 1 for finite values? Am I misreading it? If it's intentional, add a comment.